File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_LoginSessions.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_LoginSessions } from './Sys_LoginSessions.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_LoginSessionsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'LoginSession_Id', 'User_Id', 'HostName', 'IPAddress', 'NOS_UserName', 'NOS_DomainName', 'ProcessId', 'LoginTime', 'LogoutTime'];

    List: Sys_LoginSessions[] | undefined;
    ListFilter: Sys_LoginSessions[] | undefined;
    FormData!: Sys_LoginSessions;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_LoginSessions";
    }
}

File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_LoginSessionsService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_LoginSessionsService : BaseService<Sys_LoginSessions, ISys_LoginSessionsRepository>
    , ISys_LoginSessionsService
    {
    private readonly ISys_LoginSessionsRepository _Sys_LoginSessionsRepository;
    public Sys_LoginSessionsService(ISys_LoginSessionsRepository Sys_LoginSessionsRepository) : base(Sys_LoginSessionsRepository)
    {
    _Sys_LoginSessionsRepository = Sys_LoginSessionsRepository;
    }
    }
    }

